### ISSUE TITLE
Need Help Removing Joystick Inputs

### ISSUE THREAD

**Issue by sohsoONSPOTIFY** [AlanChatham/UnoJoy#70]
Hey Hey! First post of GitHub leshgo!

I recently followed a tutorial on how to set up my Arduino UNO Rev3 Clone as a arcade stick using UnoJoy.
Thus far I have been somewhat successful, binding a joystick for d-pad controls, and 6 out of 8 buttons (I have A, B, X, Y, R1, L1 bound, with start and back unbound yet.

However, when switching between Arduino and Joystick using the respective TurnIntoA... Files to test both in game input and visualised input using the ProcessingVisualiser, the joysticks are always in a top left diagonal position meaning that in game (Tekken 8 in this case), the joystick input cycles upwards through the menu constantly. Basically, I was wondering if there was a way to remove the JoyStick Inputs entirely focusing solely on the buttons and inputs I have currently set up.

I am using the default UnoJoy Files,

TL/DR: Is there any way to remove joystick input in UnoJoy?

**Comment by sohsoONSPOTIFY** [comment]
Hey all! So I did find a fix, by using a joystick module that came with my arduino kit plugged into pin A0 and A1 for Left stick X/Y axis. This allows me to use the joystick to navigate the menu, as well as using the click as a temporary back button as well! I still haven't figured out how to remove the sticks entirely, but I think this actually works out better, and I'm going to look at picking up another module for the right stick!

Therefore, the arcade stick I made is fully functional now!

![IMG_0613](https://github.com/user-attachments/assets/b6b9b7a4-28c8-4df2-9aff-e9d2ee5fb844)
![IMG_0614](https://github.com/user-attachments/assets/cc9cce10-9c70-4b18-af4f-a60ff2e5605f)
(Ignore the loose wires, one is a loose ground for the buttons, and the other two yellow ones are the input for back and start buttons which I need to replace (They are toggle buttons)).
